test/sonarplanet.test.ts

```diff
@@ -0,0 +1,27 @@
+// /// <reference types="jest" />
+
+import * as SonarPlanetAPI from "../src/sonarplanetAPI"
+
+
+describe('SonarPlanet API', () => {
+
+  it('getBrowserId returns a unique id, thx to sessionStorage', done => {
+    let idSet = new Set()
+    for (var i = 0; i < 4000; i++) {
+      idSet.add(SonarPlanetAPI.getBrowserId())
```

Comment:
You should also test the content of the (mocked) localStorage: empty at the beginning, with only one value at the end.

Reply:
Good idea!

Reply:
Done!